### PR TITLE
Network Tab Maximum Granularity.

### PR DESCRIPTION
**Pull request by sinetek**
Do not try to display network volume in gigabytes. Rather stick to a maximum size granularity of MB.

See the issue associated with this for discussion.
https://github.com/dogecoin/dogecoin/issues/2523.

### CODE REVIEW

**Comment by chromatic** [comment]
I wrote the original code, and this PR looks like it does exactly what it says it will do.

My only quibble is that it changes a MB from 1024 x 1024 bytes to 1,000,000 bytes.

**Comment by chromatic** [comment]
I built and tested this and it works as advertised. ACK.

(I still have the nit that I prefer using 1024 instead of 1000 as the divisor here, but for purely visual display it probably doesn't matter.)

**Comment by patricklodder** [comment]
> (I still have the nit that I prefer using 1024 instead of 1000 as the divisor here, but for purely visual display it probably doesn't matter.)

I was going to agree with you but after triple-checking, the current convention in the code is:
- `GB`/`MB`: based on divisors of `1000`
- `GiB`/`MiB`: based on divisors of `1024`

So if we call it `MB`, we should base it on a `1000` divisor to stay consistent.

**Comment by chromatic** [comment]
Far be it from me to argue with consistency. Full approval from me then. (Process-wise, this needs core dev review, not just core maintainer.)

**Review by patricklodder — APPROVED** [review]
tested ACK (on Intel macOS Monterey)

**Comment by patricklodder** [comment]
now blocked by #2882

**Pull request merged**
